### ISSUE TITLE
City of Zurich, Switzerland

### ISSUE THREAD

**Issue by vwmaps** [stevage/OpenTrees#28]
* Name of town/city/whatever:
* Page with information about the dataset: https://data.stadt-zuerich.ch/dataset/geo_baumkataster
* URL that directly downloads the dataset: https://data.stadt-zuerich.ch/dataset/geo_baumkataster
* Dataset format (should be CSV, GeoJSON or zipped Shapefile): all
* Country: Switzerland
* Important fields (if not in English/French - especially height, diameter, crown width): Many attributes! I'm happy to make a data dictionary for you, Steve. LMK.

**Comment by stevage** [comment]
I think I looked at this previously. You sure that is a direct download?

**Comment by mattcen** [comment]
It's not. This is just the CKAN homepage for the dataset. The download page is at https://www.stadt-zuerich.ch/geodaten/download/Baumkataster?format=10009, but it seems to require you to supply an email address to download from there. I haven't Google Translated the site enough to work out the process for that.

**Comment by mattcen** [comment]
Oh, so when I put in an email address (I used a temporary address from temp-mail.org) and played with the settings blindly for a moment, it eventually emailed me a link to a zip containing the GeoJSON. I suspect it's a temporary URL, but in any case, here it is: https://www.ogd.stadt-zuerich.ch/geoportal_download/67b14d11-68df-11ea-b1af-001dd8b73e63.zip

**Comment by vwmaps** [comment]
Yes, with an email address you can download all the data. I understand if that makes the data ineligible. It's rich data, though!

**Comment by 1-Byte** [comment]
I built [ogd-stadt-zuerich-tracker](https://github.com/1-Byte/ogd-stadt-zuerich-tracker) which lets you download datasets without ordering on each access. It uses Github Actions to update the data on a weekly basis.

Would you be interested in a PR to integrate the data (already tested locally) or do you only want to integrate data which is provided by governments directly (which would be totally fine)?
